telemetry/influxdb/src/main/scala/io/buoyant/telemetry/influxdb/InfluxDbTelemeter.scala

```diff
@@ -0,0 +1,118 @@
+package io.buoyant.telemetry.influxdb
+
+import com.twitter.finagle.Service
+import com.twitter.finagle.http.{MediaType, Request, Response}
+import com.twitter.finagle.stats.NullStatsReceiver
+import com.twitter.finagle.tracing.NullTracer
+import com.twitter.util.{Awaitable, Closable, Future}
+import io.buoyant.admin.Admin
+import io.buoyant.telemetry.{Metric, MetricsTree, Telemeter}
+
+/**
+ * This telemeter exposes metrics data in the InfluxDb LINE format, served on
+ * an admin endpoint.  It does not provide a StatsReceiver or Tracer.  It reads
+ * histogram summaries directly off of the MetricsTree and assumes that stats
+ * are being snapshotted at some appropriate interval.
+ */
+class InfluxDbTelemeter(metrics: MetricsTree) extends Telemeter with Admin.WithHandlers {
+
+  private[influxdb] val handler = Service.mk { request: Request =>
+    val response = Response()
+    response.version = request.version
+    response.mediaType = MediaType.Txt
+    val sb = new StringBuilder()
+    val host = request.host.getOrElse("none")
```

Comment:
That's correct. read on for the gritty details:

Although we're providing an endpoint in LINE format, InfluxDB expects data to be pushed, so we need Telegraf. Telegraf has a number of input plugins, such as Prometheus, which, given a list of endpoints, collects data and automatically tags them with the source. Surprisingly, there is no input plugin specifically for LINE endpoints, so we must set the `host` tag ourselves. There is an open issue in Telegraf for this:
https://github.com/influxdata/telegraf/issues/813

Because there is no LINE input plugin in Telegraf, we rely on the `inputs.exec` input plugin, which executes arbitrary commands, like this:
https://github.com/linkerd/linkerd-examples/pull/124/files#diff-898d1cb8b1ac4b01cdfca408c9069081R2